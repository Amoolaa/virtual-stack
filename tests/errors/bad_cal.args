"tests/errors/bad_cal.x2017"
