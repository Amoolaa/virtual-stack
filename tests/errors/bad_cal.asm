FUNC LABEL 0
    MOV REG 0 VAL 3
    CAL REG 0
    RET
FUNC LABEL 3
    PRINT REG 0
    RET
