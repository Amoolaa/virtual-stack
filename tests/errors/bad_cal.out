CAL: first argument must be VAL type.
